resources/metadata/sonar.yaml

```diff
@@ -1,21 +1,23 @@
 metadata:
   name: sonarExecuteScan
   description: Executes the Sonar scanner
-  longDescription: The step executes the [sonar-scanner](https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner) cli command to scan the defined sources and publish the results to a SonarQube instance.
+  longDescription: "The step executes the [sonar-scanner](https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner)
+    cli command to scan the defined sources and publish the results to a SonarQube instance."
 spec:
   inputs:
     params:
       - name: instance
         type: string
-        description: "Jenkins only: The name of the SonarQube instance defined in the Jenkins settings. DEPRECATED: use host parameter instead"
+        description: "Jenkins only: The name of the SonarQube instance defined in the Jenkins settings.
+          DEPRECATED: use host parameter instead"
```

Comment:
Is this indent necessary?

Reply:
It's actually due to line wrapping, how does this look like in the generated docs?

Reply:
See below